ATM.py

```diff
@@ -0,0 +1,120 @@
+__author__ = 'Adi Kadar Levi'
```

Comment:
please create an atm class. This class should contain all the data and functions

Reply:
done